scripts/enrich.py

```diff
@@ -56,7 +56,7 @@ def compare_titles(s1, s2):
     s1_clean = "".join(i for i in s1_uni if i.isalnum())
     s2_clean = "".join(i for i in s2_uni if i.isalnum())
 
-    return s1_clean == s2_clean
+    return Levenshtein_distance(s1_clean, s2_clean, max_distance)
```

Comment:
```suggestion
    return levenshtein_distance(s1_clean, s2_clean, max_distance)
```
We use snake_case for function names (PEP 8), sorry haha

Reply:
hehe, one of those things that is hard if the first standard you learned with did it different :)